unsteady/test_cases/trench_1d_der/moving_fn.py

```diff
@@ -0,0 +1,118 @@
+"""
+Migrating Trench Test case
+=======================
+
+Solves the hydro-morphodynamic simulation of a migrating trench using mesh movement methods
+"""
```

Comment:
Mention how this script differs from `trench_1d`

Reply:
have added some text